src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -363,11 +363,18 @@ const SourceRow = ({ connectionName, source, sourceType, networks, nodeDevices,
                              validated={validationStateSource}>
                     {installationSource}
                 </FormGroup>
-                : <OSRow os={os}
-                         osInfoList={osInfoList.filter(os => os.treeInstallable)}
+                : <>
+                    <OSRow os={os}
+                         osInfoList={osInfoList.filter(os => os.treeInstallable || (os.shortId.startsWith("rhel") && !os.version.endsWith("unknown")))}
```

Comment:
`os.shortId.startsWith("rhel")` is done in three places, and this feels a little heuristic.. Doesn't osinfo-db tell this? ("needs token" flag or so). If not, please move this into a helper function like `needsRHToken(os)` or so, so that we can see more easily where it is used, and tweak it.

What's with the "unknown" test here?